tools/ui-components/src/index.ts

```diff
@@ -1,3 +1,4 @@
 // Use this file as the entry point for component export
+import './global.css';
```

Comment:
@ahmadabdolsaheb was this the trick we were missing?

Reply:
no in the tailwind config, I did not account for TSX files.